core/domain/topic_domain_test.py

```diff
@@ -36,8 +36,8 @@ class TopicDomainUnitTests(test_utils.GenericTestBase):
 
     topic_id = 'topic_id'
 
-    def setUp(self) -> None:
-        super(TopicDomainUnitTests, self).setUp()
+    def setUp(self):
+        super().setUp()
```

Comment:
Do not remove `-> None` ditto below

Reply:
> Do not remove `-> None` ditto below

srry I am not able to understand @vojtechjelinek

Reply:
![image](https://user-images.githubusercontent.com/10142938/152650489-bbc8ecf6-ccab-4263-8871-75cf1b40559d.png)
You're removing the `-> None` part of the function definition.